src/store/UnipeptAnalysisStore.ts

```diff
@@ -30,11 +30,7 @@ const useUnipeptAnalysisStore = defineStore('PersistedAnalysisStore', () => {
     const customDatabases = useCustomFilterStore();
     const appState = useAppStateStore();
 
-    const _projectName = ref<string>("");
-
-    const isDemoMode = computed(() => {
-        return _projectName.value === "";
-    });
+    const isDemoMode = computed(() => project.isDemoMode);
```

Comment:
Since `isDemoMode` is already a reactive property of `project`, I don't think we need the `computed` here?